### PR TITLE
Webhook creation endpoint

### PR DESCRIPTION
**Pull request by jachro**

### CODE REVIEW

File: helm-chart/README.md [hunk 1]
```diff
@@ -9,3 +9,7 @@ To be executed in the `helm-chart` directory:
 ```bash
 helm upgrade --install -f minikube-values.yaml renku-graph renku-graph
 ```
+
+### Notice
+
+Due to security reasons (more [here](https://docs.gitlab.com/ee/security/webhooks.html)) webhooks urls on GitLab cannot point to local addresses. In order to make `renku-graph` services be able to create webhooks on GitLab automatically, `Allow requests to the local network from hooks and services` checkbox has to be enabled. The checkbox can be found at `http://192.168.99.100/gitlab/admin/application_settings/network#js-outbound-settings`.
```
Comment (rokroskar) [inline helm-chart/README.md]:
```suggestion
Due to security reasons (more [here](https://docs.gitlab.com/ee/security/webhooks.html)) webhooks urls on GitLab cannot point to local addresses. In order to make `renku-graph` services be able to create webhooks on GitLab automatically, `Allow requests to the local network from hooks and services` checkbox has to be enabled. The checkbox can be found at `<gitlab-endpoint>/admin/application_settings/network#js-outbound-settings`.
```

File: webhook-service/conf/routes [hunk 2]
```diff
@@ -1,3 +1,4 @@
 GET     /ping                 ch.datascience.webhookservice.HealthCheckController.ping
 
-POST    /webhook-event        ch.datascience.webhookservice.WebhookController.processWebhookEvent
\ No newline at end of file
+POST    /webhook-event        ch.datascience.webhookservice.PushEventConsumer.processPushEvent
+POST    /projects/:id/hooks   ch.datascience.webhookservice.hookcreation.HookCreationEndpoint.createHook(id: ProjectId)
```
Comment (rokroskar) [inline webhook-service/conf/routes]: newline?

File: webhook-service/README.md [hunk 3]
```diff
@@ -23,6 +23,22 @@ Verifies service health.
...
+
+A `PRIVATE-TOKEN` with user's GitLab token is required.
+
+**Response**
+
+| Status                     | Description                                                    |
+|----------------------------|----------------------------------------------------------------|
+| CREATED (201)              | For valid payloads                                             |
+| UNAUTHORIZED (401)         | When there is no `PRIVATE-TOKEN` in the header or it's invalid |
+| BAD_GATEWAY (502)          | When there were problems with webhook creation                 |
```
[the remark this answers is not in the document]
Comment (jachro) [reply webhook-service/README.md]: I went for 502 as this is endpoint is just a relay endpoint. However, I agree, giving this info to clients won't make them happier. So I changed that to 500.

File: webhook-service/conf/routes (hunk 2 above)
Comment (jachro) [reply webhook-service/conf/routes]: Done

**Pull request merged**